Tests/SentryTests/Integrations/SentryUIViewControllerSwizzlingTests.swift

```diff
@@ -0,0 +1,71 @@
+import XCTest
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+
+class SentryUIViewControllerSwizzlingTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        let options = Options()
+        let imageName = String(
+            cString: class_getImageName(SentryUIViewControllerSwizzlingTests.self)!,
+            encoding: .utf8)! as NSString
+        options.add(inAppInclude: imageName.lastPathComponent)
+        SentryUIViewControllerSwizziling.start(with: options, dispatchQueue: TestSentryDispatchQueueWrapper())
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        SentryUIViewControllerSwizziling.start(with: Options(), dispatchQueue: TestSentryDispatchQueueWrapper())
+    }
+
+    func testShouldSwizzle_TestViewController() {
+        let result = SentryUIViewControllerSwizziling.shouldSwizzleViewController(TestViewController.self)
+
+        XCTAssertTrue(result)
+    }
+    
+    func testShouldNotSwizzle_NoImageClass() {
+        let result = SentryUIViewControllerSwizziling.shouldSwizzleViewController(UIApplication.self)
+
+        XCTAssertFalse(result)
+    }
+    
+    func testShouldNotSwizzle_UIViewController() {
+        let result = SentryUIViewControllerSwizziling.shouldSwizzleViewController(UIViewController.self)
+
+        XCTAssertFalse(result)
+    }
+    
+    func testViewControllerWithoutLoadView_LoadViewNotSwizzled() {
+        testLoadViewSwizzle(viewController: TestViewController.self, loadViewSwizzled: false)
+    }
+    
+    func testViewControllerWithLoadView_LoadViewSwizzled() {
+        testLoadViewSwizzle(viewController: ViewWithLoadViewController.self, loadViewSwizzled: true)
+    }
```

Comment:
h: I removed the line 133 from SentryUIViewControllerSwizziling.m but this test did not failed.
If I understand it right this should confirm that the class loadView was indeed swizzled right?

Reply:
Valid point. Thanks 🙏